### PR TITLE
Fix CLI load_controller verb

### LINKED ISSUE #555: Example 1: load_controller cli does not work
we broke example1 with #502  
`ros2 control load_controller joint_trajectory_position_controller` does not work, because

> The 'type' param was not defined for 'joint_trajectory_position_controller'.

### PR DESCRIPTION
**Pull request by christophfroehlich**
This fixes the CLI commands of example_1 with the new option introduced with https://github.com/ros-controls/ros2_control/pull/1703

Fixes #555

### CODE REVIEW

File: ros2_control_demos-not-released.jazzy.repos [hunk 1]
```diff
@@ -1 +1,5 @@
 repositories:
+  ros2_control:
+    type: git
+    url: https://github.com/pal-robotics-forks/ros2_control.git
+    version: update/ros2controlcli
```
Comment (christophfroehlich) [inline ros2_control_demos-not-released.jazzy.repos:5]:
```suggestion
```

File: ros2_control_demos-not-released.rolling.repos [hunk 2]
```diff
@@ -1 +1,5 @@
 repositories:
+  ros2_control:
+    type: git
+    url: https://github.com/pal-robotics-forks/ros2_control.git
+    version: update/ros2controlcli
```
Comment (christophfroehlich) [inline ros2_control_demos-not-released.rolling.repos:5]:
```suggestion
```

File: ros2_control_demos.jazzy.repos [hunk 3]
```diff
@@ -9,8 +9,8 @@ repositories:
     version: master
   ros2_control:
     type: git
-    url: https://github.com/ros-controls/ros2_control.git
-    version: master
+    url: https://github.com/pal-robotics-forks/ros2_control.git
+    version: update/ros2controlcli
```
Comment (christophfroehlich) [inline ros2_control_demos.jazzy.repos:13]:
```suggestion
    url: https://github.com/ros-controls/ros2_control.git
    version: master
```

File: ros2_control_demos.rolling.repos [hunk 4]
```diff
@@ -9,8 +9,8 @@ repositories:
     version: master
   ros2_control:
     type: git
-    url: https://github.com/ros-controls/ros2_control.git
-    version: master
+    url: https://github.com/pal-robotics-forks/ros2_control.git
+    version: update/ros2controlcli
```
Comment (christophfroehlich) [inline ros2_control_demos.rolling.repos:13]:
```suggestion
    url: https://github.com/ros-controls/ros2_control.git
    version: master
```

**Comment by saikishor** [comment]
@christophfroehlich should we wait for the binary release?

**Comment by christophfroehlich** [comment]
> @christophfroehlich should we wait for the binary release?

No, because the master version is neither working with the current release, or am I wrong?

**Comment by saikishor** [comment]
> > @christophfroehlich should we wait for the binary release?
> 
> No, because the master version is neither working with the current release, or am I wrong?

Well in theory, the PR: https://github.com/ros-controls/ros2_control/pull/1703 is merged already. The current failing builds are both binary builds (main and testing)

**Comment by christophfroehlich** [comment]
But I mean the 1703 is not included in the 4.17 release. And the current version of load_controller from the head of ros2_control_demos is not working with 4.17

**Comment by saikishor** [comment]
> But I mean the 1703 is not included in the 4.17 release. And the current version of load_controller from the head of ros2_control_demos is not working with 4.17

Shall we wait for the release then=

**Comment by christophfroehlich** [comment]
The part of the demo is not working at all without merging this PR :grimacing:   after merge, at least it works for users with source builds. And I can live with red CI of the binary builds.

**Pull request merged**